Add:New Space Ruin

KA8EP opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. --> Добавил новую космическую руину, Astroak

## Ссылка на предложение/Причина создания ПР
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->
https://discord.com/channels/617003227182792704/930740675350233098/1246067807679025237

## Демонстрация изменений
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->
![image](https://github.com/user-attachments/assets/a095eaf4-2779-4653-82d2-7b78fa647753)

KA8EP committed:
Add:space ruin

Den109G commented on _maps/map_files/RandomRuins/SpaceRuins/Astroak.dmm:
```diff
@@ -0,0 +1,1612 @@
+	},
+/obj/structure/window/reinforced/survival_pod{
+	name = "window";
+	dir = 2d
```
Из-за `2d` не грузит карту. Как вообще так получилось?

KA8EP committed:
Update Astroak.dmm

Den109G commented on code/datums/ruins/space.dm:
```diff
@@ -404,3 +404,11 @@
 
+/datum/map_template/ruin/space/astroak
+	id = "Astroak"
+	suffix = "astroak.dmm"
```
Нет такой карты. Есть `Astroak.dmm`, но не `astroak.dmm`. В отличии от Виндовс, Линукс (где сервер установлен) не прощает такие ошибки.

Daeberdir committed:
Brigadier's twe...

Daeberdir committed:
Brigadier's tweaks.

- Нарс-Корги занерфлен до состояния "в полтора раза сильнее корги". Больше не убивается душами култистов.
- Убрана уникальная зона, она там не нужна, только дублировала стоковые.
- Басейн мельче, с пул контролером, дающим глубину, без стекла.
- Души культистов без неполного здоровья.
- К культистскому спессьюту добавлен спессшлем.
- Стены культа заменены на фальшивки, стены деревянные заменены на неметалические версии.
- Добавлен АПЦ, чтобы открывались стеклянные дверцы (если успеть).
- Кровавые надписи заменены на кровавые следы.
- Мапхелперы дамагтюрфа и крови.
- Добавлена мусорка с топориком, что должен намекать исследователям без мезонов на разрушение стен.
- Ручные лампы, используемые для освещения, включены. Одна оставлена перед входом.
- Одна алмазная кирка убрана.
-Размер мапы теперь 40х40, чтобы избежать близкого соседства с другими руинами.
- Руину можно найти поисковиком для подов.

Daeberdir committed:
Etot malenkiy manevr budet stoit nam 1 lishniy PR.

Daeberdir commented on code/datums/ruins/space.dm:
```diff
@@ -430,3 +430,14 @@
+	cost = 2
+	allow_duplicates = FALSE
+	can_found = TRUE
+	always_place = TRUE
```
```suggestion
```

Daeberdir committed:
Rebase issues.

Den109G reviewed (commented):
Выглядит нормально.

Daeberdir committed:
Piromage Rewiew Corrections.

- Правильные тюрфы под окном и дверью.
- Чуть больше разрушаемости для разрушенной части руины.
- Неоткручиваемые занавески.
- От себя ещё приколы на руну вкинул.
